### PR TITLE
os/arch/arm: Sync Nuttx inttypes.h

### PR DESCRIPTION
**Pull request by gSahitya-samsung**
Sync inttypes.h from Nuttx to TizenRT

### CODE REVIEW

**Review by samsung-singh — APPROVED** [review]
synced inttypes.h, good to merge.

File: os/arch/arm/include/inttypes.h [hunk 1]
```diff
@@ -0,0 +1,245 @@
+/****************************************************************************
```
Comment (sunghan-chang) [inline os/arch/arm/include/inttypes.h:1]: Need to add the Samsung license

File: os/arch/arm/include/inttypes.h [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+#define SCNxFAST8   "hhx"
+#define SCNxFAST16  "hx"
+#define SCNxFAST32  "x"
+#define SCNxFAST64  "llx"
+
+#define SCNxMAX     "llx"
+#define SCNxPTR     "x"
+
+#endif /* __ARCH_ARM_INCLUDE_INTTYPES_H */
```
Comment (sunghan-chang) [inline os/arch/arm/include/inttypes.h:245]: Please check the warning.

**Review by sunghan-chang — COMMENTED** [review]
Is there the xtensa architecture for esp? If yes, let's sync it as well. Because we support esp boards.

**Comment by gSahitya-samsung** [comment]
> Is there the xtensa architecture for esp? If yes, let's sync it as well. Because we support esp boards.

@sunghan-chang It's already up-to-date

File: os/arch/arm/include/inttypes.h (hunk 1 above)
Comment (gSahitya-samsung) [reply os/arch/arm/include/inttypes.h:18]: Done

File: os/arch/arm/include/inttypes.h (hunk 2 above)
Comment (gSahitya-samsung) [reply os/arch/arm/include/inttypes.h]: Resolved

**Review by r-prabu — APPROVED** [review]

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**